spec/factories/creditors.rb

```diff
@@ -0,0 +1,6 @@
+require 'faker'
+
+FactoryBot.define do
+  Faker::Company.name
```

Comment:
`Faker::Company.name` is a value. Therefore, you need to tell FactoryBot a value for "what"

```suggestion
  name { Faker::Company.name }
```

Same goes for `url` in this file and the attributes on `spec/factories/users.rb`